pages/common/tqdm.md

```diff
@@ -1,8 +1,16 @@
 # tqdm
 
-> Create a progress bar.
+> Show progress over time of a command.
 > More information: <https://tqdm.github.io/>.
 
+- Show iterations per second and use `stdout` afterwards:
+
+`{{seq 10000000}} | tqdm | {{command}}`
+
 - Create a progress bar:
 
-`seq 10000000 | tqdm --total 10000000 --null`
+`{{seq 10000000}} | tqdm --total 10000000 | {{command}}`
+
+- Specify unit and total:
```

Comment:
```suggestion
- Create an archive out of a directory and use the file count of that directory to create a progress bar:
```
This description should aid the user to understand what the command is doing.

Reply:
@BuonOmo You can just commit this suggestion with that button under the suggestion if you agree that it's good.